src/main/scala/scalismo/io/MeshIO.scala

```diff
@@ -275,6 +317,58 @@ object MeshIO {
     mesh
   }
 
+  private def readPLY(file: File): Try[TriangleMesh[_3D]] = {
+    val plyReader = new vtkPLYReader()
+    plyReader.SetFileName(file.getAbsolutePath)
+
+    plyReader.Update()
+
+    val errCode = plyReader.GetErrorCode()
+    if (errCode != 0) {
+      return Failure(new IOException(s"Could not read stl mesh (received error code $errCode"))
```

Comment:
Not that it matters for the discussion but this should say "Could not read ply mesh" (thought to write before forgetting :))